jck/jck.mk

```diff
@@ -171,239 +171,19 @@ ifeq ($(JDK_VERSION), 11)
-   COMPILER_LANG_STMT_TESTS_GROUP3=$(Q)lang/STMT/stmt096;lang/STMT/stmt097;lang/STMT/stmt098;lang/STMT/stmt099;lang/STMT/stmt100;lang/STMT/stmt101;lang/STMT/stmt102;lang/STMT/stmt103;lang/STMT/stmt104;lang/STMT/stmt105;lang/STMT/stmt106;lang/STMT/stmt107;lang/STMT/stmt108;lang/STMT/stmt109;lang/STMT/stmt110;lang/STMT/stmt111;lang/STMT/stmt112;lang/STMT/stmt113;lang/STMT/stmt114;lang/STMT/stmt115;lang/STMT/stmt116;lang/STMT/stmt117;lang/STMT/stmt118;lang/STMT/stmt119;lang/STMT/stmt120;lang/STMT/stmt121;lang/STMT/stmt122;lang/STMT/stmt123;lang/STMT/stmt124;lang/STMT/stmt125;lang/STMT/stmt126;lang/STMT/stmt127;lang/STMT/stmt128;lang/STMT/stmt129;lang/STMT/stmt130;lang/STMT/stmt131;lang/STMT/stmt132;lang/STMT/stmt133;lang/STMT/stmt134;lang/STMT/stmt135;lang/STMT/stmt136;lang/STMT/stmt137;lang/STMT/stmt138;lang/STMT/stmt139;lang/STMT/stmt140;lang/STMT/stmt141;lang/STMT/stmt142;lang/STMT/stmt143$(Q)
-   COMPILER_LANG_STMT_TESTS_GROUP4=$(Q)lang/STMT/stmt144;lang/STMT/stmt145;lang/STMT/stmt146;lang/STMT/stmt147;lang/STMT/stmt148;lang/STMT/stmt149;lang/STMT/stmt150;lang/STMT/stmt151;lang/STMT/stmt152;lang/STMT/stmt153;lang/STMT/stmt154;lang/STMT/stmt155;lang/STMT/stmt156;lang/STMT/stmt157;lang/STMT/stmt158;lang/STMT/stmt160;lang/STMT/stmt161;lang/STMT/stmt162;lang/STMT/stmt163;lang/STMT/stmt164;lang/STMT/stmt165;lang/STMT/stmt169;lang/STMT/stmt170;lang/STMT/stmt171;lang/STMT/stmt172;lang/STMT/stmt173;lang/STMT/stmt174;lang/STMT/stmt175;lang/STMT/stmt176;lang/STMT/stmt177;lang/STMT/stmt178;lang/STMT/stmt179;lang/STMT/stmt180;lang/STMT/stmt183;lang/STMT/stmt184;lang/STMT/stmt185;lang/STMT/stmt186;lang/STMT/stmt188;lang/STMT/stmt190;lang/STMT/stmt192;lang/STMT/stmt195;lang/STMT/stmt196;lang/STMT/stmt199;lang/STMT/stmt200;lang/STMT/stmt202;lang/STMT/stmt203;lang/STMT/stmt204;lang/STMT/stmt205$(Q)
-   COMPILER_LANG_STMT_TESTS_GROUP5=$(Q)lang/STMT/stmt206;lang/STMT/stmt207;lang/STMT/stmt250;lang/STMT/stmt251;lang/STMT/stmt252;lang/STMT/stmt253;lang/STMT/stmt254;lang/STMT/stmt255;lang/STMT/stmt256;lang/STMT/stmt257;lang/STMT/stmt258;lang/STMT/stmt259;lang/STMT/stmt300;lang/STMT/stmt301;lang/STMT/stmt302;lang/STMT/stmt303;lang/STMT/stmt304;lang/STMT/stmt305;lang/STMT/stmt306;lang/STMT/stmt307;lang/STMT/stmt310;lang/STMT/stmt311;lang/STMT/stmt312;lang/STMT/stmt313;lang/STMT/stmt314;lang/STMT/stmt315;lang/STMT/stmt316;lang/STMT/stmt317;lang/STMT/stmt318;lang/STMT/stmt319;lang/STMT/stmt320;lang/STMT/stmt321;lang/STMT/stmt322;lang/STMT/stmt323;lang/STMT/stmt324;lang/STMT/stmt325;lang/STMT/stmt326;lang/STMT/stmt327;lang/STMT/stmt330;lang/STMT/stmt331;lang/STMT/stmt332;lang/STMT/stmt333;lang/STMT/stmt334;lang/STMT/stmt335;lang/STMT/stmt336;lang/STMT/stmt337;lang/STMT/stmt338;lang/STMT/stmt339;lang/STMT/stmt350$(Q)
+   include $(TEST_ROOT)/jck/subdirs/jck8.mk
```

Comment:
Instead of having if block for all versions, can we use one line?
```
include $(TEST_ROOT)/jck/subdirs/jck$(JDK_VERSION).mk
```

Reply:
Yes will try suggested approach. We need to create new *.mk files for all version. 
Thanks